group_vars/cicognara_production.yml

```diff
@@ -47,7 +47,7 @@ rails_app_vars:
   - name: GOOGLE_CLIENT_ID
     value: '{{ vault_cicognara_google_client_id }}'
   - name: CICOGNARA_SOLR_URL
-    value: 'http://lib-solr.princeton.edu:8983/solr/cicognara'
+    value: 'http://lib-solr-prod4.princeton.edu:8983/solr/cicognara'
 sidekiq_worker_name: cicognara-workers
```

Comment:
Why not going through the load balancer `lib-solr8-prod.princeton.edu` ?

Reply:
Because we've had issues with that being slow in the past haven't we?

Reply:
We had with orangelight when we were pulling data from solr for the call_numbers.

Reply:
The solr_url for orangelight in the production boxes is pointing `lib-solr8-prod`